content/docs/command-reference/exp/index.md

```diff
@@ -37,6 +38,7 @@ positional arguments:
     gc                  Garbage collect unneeded experiments.
     push                Push a local experiment to a Git remote.
     pull                Pull an experiment from a Git remote.
+    clean               Cleanup dvc exp internal tempfiles.
```

Comment:
a possible followup on DVC itself: 

`dvc exp` -> `DVC experiments` (as in the `dvc exp init`)

or actually simplify both to just `experiments` (it's clear that it's DVC I think), and in some other commands we already do this `Remove local experiments.`, we don't do `Remove DVC experiments`. 

I can send a PR if that makes sense @pmrowla , let me know.

Reply:
Yeah that's fine with me @shcheklein

Reply:
https://github.com/iterative/dvc/pull/9149